mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -183,6 +185,11 @@ static cl::opt<bool> populateHostHarness(
     "ph", cl::desc("To populate host harness logic"),
     cl::value_desc("To populate host harness logic"), cl::init(false));
 
+static cl::opt<bool>
+    populateValidation("pv", cl::desc("To populate host validation logic"),
+                       cl::value_desc("To populate host validation logic"),
+                       cl::init(false));
+
```

Comment:
It is easy to specify those values for cpu convolution (no matter it is implemented via linalg, scf.for or extern C). It should be easy for gpu convolution as well, though I haven't figured out how.

Reply:
@yiqian1  Let's do it then, in this review:
 - For cpu conv, add the plumbing to `strides/dilations/padding`
 - For gpu conv, you shouldn't need to do anything because those are already fed to convolution ops as attributes.